common/fw/CMakeLists.txt

```diff
@@ -8,30 +8,32 @@ file(READ "firmware-version.h" ver)
 
 message(STATUS "Fetching recommended firmwares:")
 
+set(REALSENSE_FIRMWARE_URL "http://realsense-hw-public.s3.amazonaws.com" CACHE STRING)
```

Comment:
I thought the docstring was (annoyingly) required here.  Any idea why this works?
```cmake
set(<variable> <value>... CACHE <type> <docstring> [FORCE])
```

Reply:
Indeed, it does seem to be failing the CI here. Fixed now.